pages/profile/projects/[id].tsx

```diff
@@ -81,10 +85,18 @@ function ManageSingleProject({ }: Props): ReactElement {
           <div className="profilePageHeader">
             <div>
               <div className={'profilePageTitle'}>{project.name}</div>
-              <div style={{ marginBottom: 15 }}>{t('manageProjects:onlyEnglish')}</div>
+              <div style={{ marginBottom: 15 }}>
+                {t('manageProjects:onlyEnglish')}
+              </div>
             </div>
           </div>
-          <ManageProjects GUID={projectGUID} token={token} project={project} />
+          <div style={{ marginTop: '60px' }}>
+            <ManageProjects
+              GUID={projectGUID}
+              token={token}
+              project={project}
+            />
+          </div>
         </div>
       </UserLayout>
     ) : (
```

Comment:
Showing loader and No Project Found at the same time.

![Showing-loading-and-no-project-found-logo-at-the-same-time](https://user-images.githubusercontent.com/37466979/167351160-a724af75-bab2-432f-a212-51bc2307b071.PNG)

Reply:
it is fixed